BotModules/Budgett/index.js

```diff
@@ -1,57 +1,61 @@
 const schedule = require('node-schedule')
 const discord = require('discord.js');
 const getWednesdayMeme = require('./utils/utils');
-const {generalChatID, testMode, StagingID} = require('./config.json');
 
-let token = process.env.BUDGETT_TOKEN
+const chatID = process.env.BUDGETT_CHAT_ID;
```

Comment:
Well if you were deploying this bot on multiple discord server the chatID would just be the destination that you want the bot to send the messages. Calling it generalChatID or something like that basically is implying behavior that is specific to configuration, not the intended generic ability of the bot to post to a channel. I agree with changing the name for the memeNarc one because the behavior is specifically to move messages from other channels to the meme chat, however in this case I believe a generic name is more appropriate because the actual behavior is deployment specific.